assets/js/components/Pill/Pill.stories.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import HealthIcon from '@components/Health/HealthIcon';
+import Tooltip from '@components/Tooltip';
+
+import Pill from '.';
+
+export default {
+  title: 'Pill',
+  component: Pill,
+};
+
+export function Small() {
+  return <Pill size="sm">Small!</Pill>;
+}
+
+export function ExtraSmall() {
+  return <Pill size="xs">Extra small!</Pill>;
+}
+
+export function WithIcon() {
+  return (
+    <Pill className="bg-green-100 text-green-800 group flex items-center">
+      <HealthIcon size="base" health="passing" />
+      <span className="ml-1">Nice!</span>
+    </Pill>
+  );
+}
+
+export function Warning() {
+  return <Pill className="bg-yellow-200 text-yellow-600">Warning!</Pill>;
+}
+
+export function Critical() {
+  return <Pill className="bg-red-200 text-red-600">Critical!</Pill>;
+}
+
+export function Truncated() {
+  return (
+    <Pill
+      display="inline-block"
+      className="bg-green-100 text-green-800 truncate w-24"
+    >
+      Truncated pill text
+    </Pill>
+  );
+}
+
+export function WithTooltip() {
+  return (
+    <Pill className="bg-green-100 text-green-800 relative">
+      With tooltip!
+      <Tooltip tooltipText="Oh yeah!" />
+    </Pill>
+  );
+}
```

Comment:
```suggestion
export function WithTooltip() {
  return (
    <Tooltip tooltipText="Oh yeah!">
      <Pill className="bg-green-100 text-green-800 relative">
        With tooltip!
      </Pill>
    </Tooltip>
  );
}
```

I think this works and it's how the tooltip is supposed to use, can you try?

Reply:
I tried and didn't work as expected, but I will give a 2nd try

Reply:
If you have to waste more than 10 minutes on this we can absolutely merge it as is lol

Reply:
Nah, it doesn't work as expected. The tooltip is rendered out of place and with unwanted dimensions:
![test](https://user-images.githubusercontent.com/36370954/225543889-8f7ddd98-d616-4316-8568-19977c27f8f7.gif)

Reply:
RIP then